README.md

```diff
@@ -5,7 +5,126 @@
 
 ## Getting started
 
-TODO
+|Package| Download Link                                                                                        | Description                                                                |
```

Comment:
Because we link all the packages in a table, I think there's no need to preserve the nuget badge) at the very beginning of the document (only keep the build badge)